Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+#find_library(CUDART_LIBRARY cudart ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})   
+
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
+
+# We use CMake's built-in modules, used to look for SYCL capabilities.
+include( CheckIncludeFileCXX )
+include( CheckCXXSourceCompiles )
+include( CMakeParseArguments )
+set( CMAKE_REQUIRED_QUIET TRUE )
+
+# set(SYCL_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+find_package(SYCL REQUIRED)
+
+add_library(
+  ActsPluginSycl SHARED
+  src/Seeding/CreateSeedsForGroupSycl.cpp
+  src/Utilities/CreateQueueSycl.cpp
+  src/Utilities/ListSyclPlatforms.cpp
+  src/Utilities/DeviceSelector.cpp
```

Comment:
This is just a personal preference, but I like to also list the header files that belong to a library/executable. Certain IDEs (most notably [Visual Studio](https://visualstudio.microsoft.com/) and [Xcode](https://developer.apple.com/xcode/)) set up much nicer projects with the header files listed as well.

Reply:
ok